src/services/database/serverless.yml

```diff
@@ -0,0 +1,77 @@
+service: ${self:custom.project}-database
+
+frameworkVersion: "3"
+
+package:
+  individually: true
+
+plugins:
+  - serverless-stack-termination-protection
+  - "@stratiformdigital/serverless-s3-security-helper"
+  - "@stratiformdigital/serverless-iam-helper"
+  - serverless-bundle
+
+provider:
+  name: aws
+  region: us-east-1
+  iam:
+    role:
+      path: /delegatedadmin/developer/
+      permissionsBoundary: arn:aws:iam::${aws:accountId}:policy/cms-cloud-admin/developer-boundary-policy
+      statements:
+        - Effect: Allow
+          Action:
+            - logs:CreateLogGroup,
+            - logs:CreateLogStream,
+            - logs:PutLogEvents,
+            - dynamodb:PutItem,
+            - dynamodb:UpdateItem,
+            - dynamodb:DeleteItem"
+          Resource: "*"
+  stackTags:
+    PROJECT: ${self:custom.project}
+    SERVICE: ${self:service}
+
+custom:
+  project: ${env:PROJECT}
+  fileHash: ${file(./items.ts)}
+  serverlessTerminationProtection:
+    stages: # Apply CloudFormation termination protection for these stages
+      - master
+      - val
+      - production
+  tableName: ${self:custom.project}-${self:service}-${sls:stage}
+
+parameters:
+  fileHash:
+    Type: String
+    Default: ""
+
+functions:
+  seeddatabase:
+    handler: handlers/seeddatabase.handler
+
+resources:
+  Resources:
+    Table:
+      Type: AWS::DynamoDB::Table
+      Properties:
+        TableName: ${self:custom.tableName}
+        AttributeDefinitions:
+          - AttributeName: id
+            AttributeType: S
+        KeySchema:
+          - AttributeName: id
+            KeyType: HASH
```

Comment:
we really do want to give people both a primary key and a sort key here to help them establish solid patterns. so something like:

```
 AttributeDefinitions:
          - AttributeName: "PK"
            AttributeType: "S"
          - AttributeName: "SK"
            AttributeType: "S"
        KeySchema:
          - AttributeName: "PK"
            KeyType: "HASH"
          - AttributeName: "SK"
            KeyType: "RANGE"
            ```

Reply:
I have added a sort key
  AttributeDefinitions:
          - AttributeName: name
            AttributeType: S
          - AttributeName: age
            AttributeType: N
        KeySchema:
          - AttributeName: name
            KeyType: HASH
          - AttributeName: age
            KeyType: RANGE